src/firefly/java/edu/caltech/ipac/firefly/resources/irsa-image-master-table.csv

```diff
@@ -89,7 +89,7 @@ Spitzer,FLS: Spitzer First Look Survey,Spitzer,spitzer.fls_irac,FLS  IRAC4,IRAC4
 Spitzer,FLS: Spitzer First Look Survey,Spitzer,spitzer.fls_mips,FLS  MIPS24,MIPS24,,MIPS24 (24 microns),24,24 microns,mid-IR,extragalactic,extragalactic,0.125,0.25,http://irsa.ipac.caltech.edu/data/SPITZER/FLS/,FLS IRAC 24  microns,ATLAS,file_type='science' and fname like '%mosaic.fits',image
 Spitzer,FLS: Spitzer First Look Survey,Spitzer,spitzer.fls_mips,FLS  MIPS70,MIPS70,,MIPS70 (70 microns),70,70 microns,far-IR,extragalactic,extragalactic,0.125,0.25,http://irsa.ipac.caltech.edu/data/SPITZER/FLS/,FLS IRAC 70  microns,ATLAS,file_type='science',image
 Spitzer,FLS: Spitzer First Look Survey,Spitzer,spitzer.fls_mips,FLS  MIPS160,MIPS160,,MIPS160 (160 microns),160,160 microns,far-IR,extragalactic,extragalactic,0.125,0.25,http://irsa.ipac.caltech.edu/data/SPITZER/FLS/,FLS IRAC 160  microns,ATLAS,file_type='science',image
-Spitzer,FLS: Spitzer First Look Survey,Radio,spitzer.fls_gbt_hi,FLS  21cm,21 cm,, 21 cm,21,21 cm,radio,extragalactic,extragalactic,0.125,0.25,http://irsa.ipac.caltech.edu/data/SPITZER/FLS/,FLS GBT 21 cm,ATLAS,file_type='science' and fname like '%TBHICUBE%',image
+Spitzer,FLS: Spitzer First Look Survey,Radio,spitzer.fls_gbt_hi,FLS  210000 microns,210000 microns,, 210000 microns,210000,210000 microns,radio,extragalactic,extragalactic,0.125,0.25,http://irsa.ipac.caltech.edu/data/SPITZER/FLS/,FLS GBT 210000 microns,ATLAS,file_type='science' and fname like '%TBHICUBE%',image

```

Comment:
For example here, label = 21 cm, but min, max value = 21000, 21000.

Reply:
Oh sorry for that, issue solved!